cf-java-logging-support-opentelemetry-agent-extension/README.md

```diff
@@ -75,8 +76,8 @@ That means, without any configuration the agent with the extension will forward
 The difference between `cloud-logging` and `otlp` exporters are explained in an own [section](#implementation-differences-between-cloud-logging-and-otlp-exporter).
 The benefit of the `cloud-logging` exporter is, that it can be combined with a different configuration of the `otlp` exporter.
```

Comment:
I would assume this also applies for Dynatrace, right?

Reply:
No, there is no default configuration for the `otlp` exporter and Dynatrace. You need to use the `dynatrace` exporter explicitly. Note, that `otlp` is considered to be a legacy/fallback approach. Since Cloud Logging is always available, it is used as the default.